pkg/storage/tsdb/caching_config.go

```diff
@@ -135,14 +154,13 @@ func CreateCachingBucket(chunksConfig ChunksCacheConfig, metadataConfig Metadata
 			attributesCache = metadataCache
 		}
 		if chunksConfig.AttributesInMemoryMaxItems > 0 {
-			var err error
 			attributesCache, err = cache.WrapWithLRUCache(attributesCache, "chunks-attributes-cache", prometheus.WrapRegistererWithPrefix("cortex_", reg), chunksConfig.AttributesInMemoryMaxItems, chunksConfig.AttributesTTL)
 			if err != nil {
 				return nil, errors.Wrapf(err, "wrap metadata cache with in-memory cache")
 			}
 		}
 
-		cfg.CacheGetRange("chunks", chunksCache, isTSDBChunkFile, subrangeSize, attributesCache, chunksConfig.AttributesTTL, chunksConfig.SubrangeTTL, chunksConfig.MaxGetRangeRequests)
+		cfg.CacheAttributes("chunks-attributes", attributesCache, isTSDBChunkFile, chunksConfig.AttributesTTL)
```

Comment:
I don't think we should do this change in this PR, because this PR doesn't add the new chunks cache support (just preliminary code, but not using it yet).